src/proto/macro.proto

```diff
@@ -1,14 +1,14 @@
 syntax = "proto2";
 
-package mixx.track.io;
+package mixxx.track.io;
 
 option optimize_for = LITE_RUNTIME;
 
 message Macro {
   message Action {
     optional uint32 type = 1;
     optional uint64 origin = 2;
```

Comment:
if these are frame positions, use doubles

Reply:
oh right. Why are they doubles anyways? What is a fractional frame?

Reply:
https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/beatgrid.20editing.20UI/near/203780641

Reply:
Hmm okay, I don't see an actual consensus there though. The arguments of processing vs storage weren't refuted.